dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -125,7 +125,7 @@ xPoweredBy.name=X-Powered-By
 xPoweredBy.desc=Alternative string for the X-Powered-By header setting. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.xpoweredby. There is no default value for this property. If the property is not set, the value of the X-Powered-By header is set to Servlet/<servlet spec version>, as defined by the Servlet specification.
 
 disableXPoweredBy.name=Disable the X-Powered-By header
-disableXPoweredBy.desc=Disable setting the X-Powered-By header. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disablexpoweredby.
+disableXPoweredBy.desc=Disable setting the X-Powered-By header. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disablexpoweredby. The default is true for servlet-5.0 and newer Servlet features.
```

Comment:
is there a reason the property is not described here in terms other than it's equivalent?

Reply:
This property function is quite simple, we disable the X-Powered-By header being set by the runtime by a servlet feature. We do outline that (this is an existing property I'm just making some updates): "Disable setting the X-Powered-By header". I'm happy to add more details just not quite sure what else we should add. Thoughts? Thanks!

Reply:
ok thanks @pnicolucci - if it doesn't require users to look at the full application server profile to understand the function then this is ok.